brave/brave-core
# Show unavailable message if user is in disconnected state

Resolves https://github.com/brave/brave-browser/issues/20991

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Screenshots

<img width="413" alt="Screen Shot 2022-05-17 at 5 11 29 PM" src="https://user-images.githubusercontent.com/5995084/168912866-8c79d632-f7eb-467e-8a3a-3b87c04f4e9d.png">
<img width="392" alt="Screen Shot 2022-05-17 at 5 24 58 PM" src="https://user-images.githubusercontent.com/5995084/168912869-749ea34a-adde-4765-88f2-083202b98a65.png">
<img width="412" alt="Screen Shot 2022-05-17 at 5 25 10 PM" src="https://user-images.githubusercontent.com/5995084/168912874-ad6b4d3b-9e1f-4170-ade1-0d36e4aabe21.png">
<img width="290" alt="Screen Shot 2022-05-17 at 5 25 25 PM" src="https://user-images.githubusercontent.com/5995084/168912879-661b9c5c-b776-41d4-a33e-3e60b0945559.png">


## Test Plan

- Open browser with a fresh profile.
- Enable Rewards and verify with an external wallet provider.
- Invalidate the authentication tokens for the wallet provider.
  - For Uphold, this can be accomplished by visiting your account page, clicking the "3 dots" more menu, and selecting "Integrations". The integration for "Brave Browser" can then be removed.
- Visit the NPT and verify that "login" messaging appears.
- Open the Rewards panel and verify that "login" messaging appears.
- Visit the Rewards page and verify that "login" messaging appears.
- Verify that clicking on the login messaging box will take the user to the wallet provider authorization page.
